frontend/admin/src/js/components/user/AdminAboutSection.tsx

```diff
@@ -39,6 +42,54 @@ const AdminAboutSection: React.FC<AdminAboutSectionProps> = ({
             })}
           </p>
         )}
+        {isVeteran !== null && (
+          <p>
+            {intl.formatMessage({
+              defaultMessage: "Member of CAF:",
+              description: "Veteran/member label",
+            })}{" "}
+            <span data-h2-font-weight="b(700)">
+              {isVeteran
+                ? intl.formatMessage({
+                    defaultMessage: "Veteran or member of the CAF",
+                    description:
+                      "message admin side candidate profile, veteran status",
+                  })
+                : intl.formatMessage({
+                    defaultMessage: "Not a veteran or member of the CAF",
+                    description:
+                      "message admin side candidate profile, not veteran status",
+                  })}
+            </span>
+          </p>
+        )}
+        {citizenship ? (
+          <p>
+            {intl.formatMessage({
+              defaultMessage: "Citizenship:",
+              description: "Citizenship label",
+            })}{" "}
+            <span data-h2-font-weight="b(700)">
+              {citizenship === CitizenshipStatus.Citizen &&
+                intl.formatMessage({
+                  defaultMessage: "Canadian Citizen",
+                  description: "Canadian Citizen status",
+                })}
+              {citizenship === CitizenshipStatus.PermanentResident &&
+                intl.formatMessage({
+                  defaultMessage: "Permanent Resident",
+                  description: "permanent resident status",
+                })}
+              {citizenship === CitizenshipStatus.Other &&
+                intl.formatMessage({
+                  defaultMessage: "Other",
+                  description: "other citizenship status",
+                })}
```

Comment:
I think we have an established pattern for this in:

[@common/constants/localizedConstants](https://github.com/GCTC-NTGC/gc-digital-talent/blob/7474377cbffdb318d948d6a1c4d4590fbabed117/frontend/common/src/constants/localizedConstants.tsx)

Reply:
aye aye
https://github.com/GCTC-NTGC/gc-digital-talent/pull/3664/commits/90fa099fc8577f9db4a389f42d6a1aecb75f286c